src/bot/handlers.py

```diff
@@ -117,6 +121,51 @@ def get_last_f1_fantasy_race_standing(update: Update, context: CallbackContext):
     return get_last_f1_fantasy_race_standing
 
 
+def get_last_race_team_standing_handler(cookies: str, league_id: str):
+    def get_f1_last_race_team_standing_handler(
+        update: Update, context: CallbackContext
+    ) -> None:
+        f1_fantasy_standings_req = requests.get(
+            url=f"https://fantasy-api.formula1.com/f1/2022/leaderboards/leagues?v=1&league_id={league_id}",  # noqa: E501
+            headers={"Cookie": cookies},
+        )
+        standings = decode_http_response(f1_fantasy_standings_req, to_league_standings)
+        inputs = entrant_to_pretty_input(standings)
+        reply_markup = InlineKeyboardMarkup(inputs)
+
+        update.message.reply_text("Please choose:", reply_markup=reply_markup)
+
+    return get_f1_last_race_team_standing_handler
+
+
+def get_last_race_team_standing_handler_button(cookies: str):
+    def get_f1_last_race_team_standing_handler_button(
+        update: Update, context: CallbackContext
+    ) -> None:
+
+        query = update.callback_query
+
+        query.answer()
+        user_global_id = query.data
+
+        # WIP: find a way to get last race dinamically
+        f1_fantasy_standing_team_req = requests.get(
+            url=f"https://fantasy-api.formula1.com/f1/2022/picked_teams/for_slot?v=1&game_period_id=8&slot=1&user_global_id={user_global_id}",  # noqa: E501
```

Comment:
Hardcoded race for testing purposes. We need to refactor the last race method to make it usable everywhere